src/cov/cov-report.c

```diff
@@ -1029,6 +1032,29 @@ static void cover_print_hierarchy_guts(FILE *f, cover_report_ctx_t *ctx)
               "   function GetExclude(excludeCmd) {\n"
               "      navigator.clipboard.writeText(excludeCmd);\n"
               "   }\n"
+              "   function getCellValue (tr, n) {\n"
+              "      v = tr.getElementsByTagName(\"TD\")[n];\n"
```

Comment:
```suggestion
              "      let v = tr.getElementsByTagName(\"TD\")[n];\n"
```

Otherwise it creates a global variable.

Reply:
Thx, I am absolute Javascript rookie. The code in `cov-report` is literally the only Javascript I ever "wrote" :D